packages/core/admin/admin/src/components/UnauthenticatedLogo/index.js

```diff
@@ -11,7 +11,7 @@ const Logo = () => {
     logos: { auth },
   } = useConfigurations();
 
-  return <Img src={auth.default} aria-hidden alt="" />;
+  return <Img src={auth.custom} aria-hidden alt="" />;
```

Comment:
It is more of a  question: should this be:

```suggestion
  return <Img src={auth?.custom ?? auth.default} aria-hidden alt="" />;
```